tools/iothub-explorer/doc/provision_device.md

```diff
@@ -28,6 +28,32 @@ Created device mydevice
 ```
 Copy the device connection string information for later use.
 
+## Provision multiple devices in your IoT hub
+
+The iothub-explorer tool also enables you to provision multiple new devices at once. To provision multiple new devices:
+
+1. Create a text file and place the device id of each device you want to provision on a new line within the text file.
+2. Get the connection string for your IoT Hub. See [Setup IoT Hub][setup-iothub] for more details.
+3. Run the following commands in a terminal/shell window from the tools/iothub-explorer folder in the repository using the IoT Hub connection string from the previous step and replacing "mydevice" with your desired device name.
```

Comment:
"...using the IoT Hub connection string and text file path from the previous steps."  The bit about "mydevice" doesn't apply here.